subsys/bluetooth/host/conn_internal.h

```diff
@@ -126,8 +126,13 @@ struct bt_conn_iso {
 	};
 
 #if defined(CONFIG_BT_ISO_UNICAST) || defined(CONFIG_BT_ISO_BROADCASTER)
-	/** 16-bit sequence number that shall be incremented per SDU interval */
-	uint16_t seq_num;
+	/** @brief 16-bit sequence number that shall be incremented per SDU interval
+	 *
+	 *  We store it as 32-bit to handle wrapping: Only once the value has
```

Comment:
Nitpicking: Remove "we"?
"We store it" -> "Stored"
"will we allov [...]" -> "will values less than the current be allowed again"